tests/lib/rules/order-in-components.js

```diff
@@ -321,6 +321,15 @@ eslintTester.run('order-in-components', rule, {
       });`,
       parserOptions: { ecmaVersion: 6, sourceType: 'module' },
     },
+    {
```

Comment:
Could you please also add one failing test with UnaryExpression placed in the wrong place?